qa_forum/static/js/main.js

```diff
@@ -0,0 +1,39 @@
+$(document).ready(function(){
+
+    $('.comment_submit').on('click', function() {
+        var raiser = $(this).prev();
```

Comment:
@chuckcwh Why keep calling `$(this)`? Maybe it's better to call it just once and store it in a variable?